test/deploy/kubernetes/deploy.sh

```diff
@@ -99,6 +116,23 @@ kubectl create configmap solution-config \
     --from-file="$BASEPATH"/solution-config/config.yaml \
     --namespace "$NAMESPACE";
 
+# Create Config Map For Machine-id
+kubectl create configmap control-node-id --from-literal=id=9995f539f4f770e2a3fe9e2e615c32a8 --namespace "$NAMESPACE";
+if [ $REDEFPODS = true ]; then
+    kubectl create configmap ha-node-id --from-literal=id=1115f539f4f770e2a3fe9e2e615c32a8 --namespace "$NAMESPACE";
+    kubectl create configmap server-node1-id --from-literal=id=ddd120a9e051d103c164f605615c32a4 --namespace "$NAMESPACE";
```

Comment:
What's use of these, we nywaz have hard-coded values in pod definition files???

Reply:
We have shifted from Pod to Deployment kind so we are not using downward API for placing machine id inside containers, Instead now we're using configmap (Same as like services team is using.)